aten/src/ATen/native/native_functions.yaml

```diff
@@ -3035,6 +3035,14 @@
   variants: method
   device_guard: False
 
+- func: vander(Tensor x, *, bool increasing=False) -> Tensor
```

Comment:
why don't we combine these schemas like in the documentation?  I think that would be

`vander(Tensor x, int? n=None, bool increasing=False) -> Tensor`